index.js

```diff
@@ -9,11 +18,86 @@ wrapId.addEventListener('click', () => {
   rightNavebarId.classList.toggle('change-rightnavbar');
 });
 
+const popupArray = [{
+  Name: 'Keeping track of hundreds  of components website',
+  Name1: 'Multi Post Stories',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+  featureimage: './images/popupimage.png',
+  technologies: ['html', 'Bootstrap', 'Ruby on rails'],
+  linktoliveversion: '#',
+  linktosource: '#',
+}];
```

Comment:
- [ ] As requested by the previous reviewer, each project must have an object of its full details stored in this array. When we click for example on the button of the project we must only see the related information of that project. We can not store only one project object because you will need to add different projects with different details. Please each project has its own object information and must be rendered when the `see Project` button is clicked.